### ISSUE TITLE
Implement the remainder of the C_Addon API

### ISSUE THREAD

**Issue by rdw-software** [rdw-archive/WebClient#15]
From the original prototype:

```
isAddonEnabled(addonName) {},
enableAddon(addonName) {},
disableAddon(addonName) {},
enableAllAddons() {},
disableAllAddons() {},
getAddonInfo(addonName) {},
getAddonMetadata(addonName) {},
isOutOfDateAddon(addonName) {},
```

It's basically all of the WOW Addon API functionality, which may be overkill at this point.

**Comment by rdw-software** [comment]
Bumping this up as I've prototyped most of the API already and some parts are needed for https://github.com/RevivalEngine/WebClient/issues/65.

**Comment by rdw-software** [comment]
Resolved as of 935154ff4a715fbf1e2c45f3a526bbb3defb4cfb. The out-of-date stuff is a separate issue #72.
